pkgs/tools/networking/openconnect/default.nix

```diff
@@ -1,8 +1,14 @@
-{ stdenv, fetchurl, pkgconfig, vpnc, openssl ? null, gnutls ? null, gmp, libxml2, stoken, zlib } :
+{ stdenv, fetchurl, pkgconfig, openssl ? null, gnutls ? null, gmp, libxml2, stoken, zlib, fetchgit, darwin } :
 
 assert (openssl != null) == (gnutls == null);
 
-stdenv.mkDerivation rec {
+let vpnc = fetchgit {
```

Comment:
Why is this different from the vpnc package? If possible it would be better to use `vpnc.src` instead so it gets updated consistently.

Reply:
openconnect recommend this version over the script in the original vpnc package: 
> this updated version [...] has support for IPv6, and for running on Solaris and on newer Linux kernels amongst other bug fixes.

(ref [openconnect website](https://www.infradead.org/openconnect/vpnc-script.html))